From 24c1cea3bb964d6d2255c2bd9f1433413aa3e694 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Tue, 30 Jul 2013 06:45:48 +0200
Subject: [PATCH] sanitize system clucene includes

Apparently clucene bundles pieces of a very old version of boost (1.42)
in CLucene/ext/boost, which breaks build with boost 1.54.

(cherry picked from commit 24add548ac163651de5e3372798f0d4db347de77)
Signed-off-by: David Tardon <dtardon@redhat.com>

Conflicts:
	configure.ac

Change-Id: Ib244222f8f048aefe0484060f9ad5f41bfd24def
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 03d2ef8..ce04c65 100644
--- a/configure.ac
+++ b/configure.ac
@@ -7144,6 +7144,7 @@ if test "$with_system_clucene" = "yes"; then
     AC_MSG_RESULT([external])
     SYSTEM_CLUCENE=YES
     PKG_CHECK_MODULES(CLUCENE, libclucene-core)
+    CLUCENE_CFLAGS=[$(printf '%s' "$CLUCENE_CFLAGS" | sed -e 's@-I[^ ]*/CLucene/ext@@')]
     AC_LANG_PUSH([C++])
     save_CXXFLAGS=$CXXFLAGS
     save_CPPFLAGS=$CPPFLAGS
-- 
1.8.3.1

